src/commands/weekly.ts

```
@@ -40,6 +40,7 @@ export const weeklyCommand = define({
 			order: ctx.values.order,
 			offline: ctx.values.offline,
 			startOfWeek: ctx.values.startOfWeek,
+			locale: ctx.values.locale,
```

ESLint

[error] Unsafe assignment of an `any` value.

(ts/no-unsafe-assignment)